tests/validation/tests/v3_api/test_bkp_restore_local.py

```diff
@@ -22,6 +24,131 @@ def test_bkp_restore_local_delete():
     validate_backup_delete(ns, binfo)
 
 
+@if_test_rbac
+def test_rbac_create_bkp_restore_local_cluster_owner():
+    """ Only cluster-owner should be allowed to create backups """
+    user_token = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    user_client = get_client_for_token(user_token)
+    user_cluster = user_client.list_cluster(name=CLUSTER_NAME).data[0]
+    backup = user_cluster.backupEtcd()
+    backupname = backup['metadata']['name']
+    wait_for_backup_to_active(user_cluster, backupname)
+    assert len(user_cluster.etcdBackups(name=backupname)) == 1
+
+
+@if_test_rbac
+@pytest.mark.parametrize("role", rbac_roles)
```

Comment:
We could add if statement here if role == "CLUSTER_OWNER" else; instead of having 2 tests? 
Same for the other methods also:
test_rbac_list_bkp_restore_local_cluster_owner

Reply:
Resolved as per our QA call.